scripts/create-triggers-accounts.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
+set -ex
+
+#############################################
```

Comment:
What would be extremely useful is a bit of info about how this test (and subsequent tests) works. 

IIUC what you are doing here is creating a copy of the accounts table called _new_accounts and keeping both tables synchronized using triggers. The triggers will update the _new_accounts table with any modifications that occur while the migration is running.

To check that the triggers work correctly, you create the _new_accounts table, and then the accounts table. On the accounts table, you add the triggers necessary to synchronize the tables. Then you populate the accounts table using the unit tests, which should cause the triggers to fire and the _new_accounts table to reflect those changes. Once data population script runs, you run the big UNION statement to ensure the data in both tables is the same.

The end goal for the _new_accounts table is to provide a copy of the accounts table and any modifications that occur during the migration. The _new_accounts table can then be used as a known good reference data set to check the migrated data against.

Is that an accurate, if somewhat inelegant, summary?